dbos/_client.py

```diff
@@ -364,3 +367,6 @@ async def fork_workflow_async(
             fork_workflow, self._sys_db, self._app_db, workflow_id, start_step
         )
         return WorkflowHandleClientAsyncPolling[R](forked_workflow_id, self._sys_db)
+
+    def reset_system_database(self, database_url: URL, sysdb_name: str) -> None:
```

Comment:
Why is this here?  It's a client method that doesn't use the client instance variables and will actually silently destroy the client.